x-pack/plugins/alerting/server/rules_client/tests/bulk_delete.test.ts

```diff
@@ -313,6 +353,21 @@ describe('bulkDelete', () => {
     );
   });
 
+  test('should not mark API keys for invalidation if the user is authenticated using an api key', async () => {
+    unsecuredSavedObjectsClient.bulkDelete.mockResolvedValue({
+      statuses: [{ id: 'id3', type: 'alert', success: true }],
```

Comment:
can we add id1 and id2 here as well, ensure that they get passed to the bulkKeyInvalidation function

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/154189/commits/bc37396a966b90e9a37fc2b1039be91dd1403c89